### DISCUSSION TITLE
Can you import new features from `build` project?

### DISCUSSION THREAD

**Discussion by ghost** [premake/premake-core#2307] [Ideas]
This is RudjiGames' `build` (https://github.com/RudjiGames/build). It's intended to use with a Premake4 fork called GENie. It's the build system of RudjiGames' MTuner memory profiler (https://github.com/RudjiGames/MTuner). `build` provides various features, most notably is Qt support (MTuner is a Qt application).

**Comment by Jarod42** [comment]
[premake-qt](https://github.com/dcourtois/premake-qt) might interest you.
